apps/condo/domains/meter/access/_internalDeleteMeterAndMeterReadingsService.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Generated by `createservice meter._internalDeleteMeterAndMeterReadingsService --type mutations`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+async function can_internalDeleteMeterAndMeterReadings ({ authentication: { item: user } }) {
```

Comment:
we do not use underscore in naming

Reply:
this is only for internal mutations (_internal)

Reply:
`can_internalSomething` seems strange. Even if mutation name contains underscore, not sure that access should contain it like this :)

Reply:
Just because someone else did it doesn't mean it's right. The underline in the middle of the camelCase name is odd.